packages/cosmic-swingset/lib/ag-solo/main.js

```diff
@@ -63,7 +63,7 @@ start
     const { webhost, egresses } = subOpts;
     const basedir = subArgs[0] || AG_SOLO_BASEDIR;
     const subdir = subArgs[1];
-    insist(basedir !== undefined, 'you must provide a BASEDIR');
+    assert(basedir !== undefined, 'you must provide a BASEDIR');
```

Comment:
Does this version, without `details` work?

Reply:
It does. This was because of making it backwards compatible with the `assert` APIs. It probably means we will eventually want an eslint rule to just use the `details` version to protect any sensitive data.